web-app/src/common/prediction.spec.ts

```diff
@@ -114,4 +114,37 @@ describe('Get a suggestion for the next play', () => {
 
     expect(() => getNextPlaySuggestion(input, "white")).toThrowError();
   });
+
+  it('Should get a suggestion to play on x:3,y:1', () => {
+    const input = parseGameStateFromMultilineString(`
+    ⬡ ⬡ ⬡ ⬡ ⬡
+     ⬡ ⬡ ⬡ ⬡ ⬡
+      ⬡ ⬡ ⬢ W ⬡
+       ⬡ ⬢ ⬡ W ⬡
+        ⬡ ⬡ ⬡ ⬡ ⬡
+    `);
+
+    const nextPlayAdvice = getNextPlaySuggestion(input, "white");
+
+    expect(nextPlayAdvice).toStrictEqual({ x: 3, y: 1 });
+  });
+
+  it('Should get a suggestion to play on one from 3 possible coordinates', () => {
+    const input = parseGameStateFromMultilineString(`
+    ⬡ W ⬡ ⬡ ⬡
+     ⬡ ⬡ ⬡ ⬡ ⬢
+      W ⬡ ⬢ W ⬡
+       ⬡ ⬢ ⬡ W ⬡
+        ⬡ ⬡ ⬡ ⬡ ⬡
+    `);
+
+    const nextPlayAdvice = getNextPlaySuggestion(input, "black");
+
+    expect(areExpectedCoordinatesInList(nextPlayAdvice, [

```

Comment:
I didn't see it before, but no need to define a function for that :
```expect([
    { x: 3, y: 1 },
    { x: 0, y: 3 },
    { x: 0, y: 4 },
]).toContainEqual(nextPlayAdvice)```

Advantage : 
- Test report is clearer
- No utils method to maintain
- The method from jest expect is bullet proof (yours is not...)

In general, take some time to find if the assertion library can check what you want to, before adding a custom one.